test/language/expressions/tagged-template/invalid-escape-sequences.js

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2016 Tim Disney. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: Invalid unicode escape sequence in tagged template literals are allowed
+es6id: 11.8.6
+---*/
+
+(strs => {
+  assert(strs[0] === undefined);
+  assert(strs.raw[0] === '\\01');
+})`\01`;
```

Comment:
can we use `assert.equal` and provide an assertion message? Otherwise failing tests would be very confusing in this file.